fgd/point/momentum/momentum_weapon_spawner.fgd

```diff
@@ -0,0 +1,8 @@
+@PointClass base(MomentumPickup)
+	appliesto(MOMENTUM)
+	size(-43 -43 -36, 43 43 66)
+= momentum_weapon_spawner : "Weapon Spawner"
+	[
+	WeaponName(string) : "Weapon Name" : "" : "The name of the weapon to give the player."
```

Comment:
It would probably be better if this was a `choices` type key value, as it wouldn't be immediately clear what is a valid weapon name